### DISCUSSION TITLE
Ignore all-day events

### DISCUSSION THREAD

**Discussion by 30ka** [derekantrican/GAS-ICS-Sync#350] [Q&A] [answered]
I am trying to find a way to make the script ignore all-day events but I am in way over my head.
Something like https://github.com/derekantrican/GAS-ICS-Sync/issues/263 but exactly inverse.

Anybody willing to help a brother out? 

Thank you for your time!

**Comment by jonas0b1011001** [comment, marked as answer]
You can invert #263 by replacing `==` with `!=`.

```
result = result.filter(function(event){
    return (event.getFirstPropertyValue('dtstart').toString().indexOf("T") != -1);
  });
```

**Reply by derekantrican** [reply]
@30ka If this works for you, please "Mark as answer"

**Reply by 30ka** [reply]
This is genious. Works perfectly. Thank you! :)
